src/Symfony/Component/PropertyAccess/StringUtil.php

```diff
@@ -203,7 +203,7 @@ public static function singularify($plural)
         }
 
         // Convert teeth to tooth, feet to foot
-        if (false !== ($pos = strpos($plural, 'ee')) && strlen($plural) > 3 && 'feedback' !== $plural) {
+        if (false !== ($pos = strpos($plural, 'ee')) && strlen($plural) > 3 && !preg_match('{feedback|Feedback|career|Career}', $plural)) {
```

Comment:
This class responsibility is to find a plural or a singular of a word, feedBacktrace or careerSubject  are not words. I'm sure I'm missing something here.